autode/hessians.py

```diff
@@ -400,13 +389,42 @@ def frequencies_proj(self) -> List[Frequency]:
                 "Could not calculate projected frequencies, must "
                 "have atoms set"
             )
-
         n_tr = self.n_tr  # Number of translational+rotational modes
-        lambdas = np.linalg.eigvalsh(self._proj_mass_weighted[n_tr:, n_tr:])
 
-        trans_rot_freqs = [Frequency(0.0) for _ in range(n_tr)]
+        H = self._proj_mass_weighted
```

Comment:
I think this whole part is not needed, and the original code was fine(?). Projecting the Hessian should mean that we are safe to assume the first 5 or 6 columns to be zero. If they are not zero - then that is probably the measure of how much numerical error there is in the Hessian. (btw, I have been trying to understand the math - and so far I have been able to mathematically prove that the first 5 / 6 columns of the projected Hessian definitely have to be zero. For the first 5 / 6 rows it also makes intuitive sense - but I haven't been able to find the mathematical proof yet)